src/senaite/ast/upgrade/v01_01_000.py

```diff
@@ -45,3 +50,42 @@ def upgrade(tool):
 
     logger.info("{0} upgraded to version {1}".format(PRODUCT_NAME, version))
     return True
+
+
+def fix_wrong_results_resistance(tool):
+    """Walks thorugh "to_be_verified" AST resistance tests and forces the
+    update of the result if it's current result is '-' or 'NA'
+    """
+    logger.info("Fix wrong AST results ...")
+    query = {
+        "portal_type": "Analysis",
+        "review_state": "to_be_verified",
+        "getKeyword": RESISTANCE_KEY,
+    }
+    brains = api.search(query, ANALYSIS_CATALOG)
+
+    total = len(brains)
+    for num, brain in enumerate(brains):
+        if num and num % 100 == 0:
+            logger.info("Processed objects: {}/{}".format(num, total))
+
+        try:
+            obj = api.get_object(brain, default=None)
+        except AttributeError:
+            obj = None
+
+        if not obj:
+            uncatalog_brain(brain)
```

Comment:
Move this into the except block and continue, otherwise the `obj.getResult()` in the next line will fail

Reply:
I've just added continue after the `uncatalog_brain`